src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -135,6 +135,27 @@
   <data name="Settings_FileName_Cannot_Be_A_Path" xml:space="preserve">
     <value>Parameter 'fileName' to Settings must be just a fileName and not a path</value>
   </data>
+  <data name="ShowError_ConfigInvalidOperation" xml:space="preserve">
+    <value>NuGet.Config is malformed, Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="ShowError_ConfigInvalidXml" xml:space="preserve">
+    <value>NuGet.Config is invalid XML, Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="ShowError_ConfigRootInvalid" xml:space="preserve">
+    <value>NuGet.Config is invalid, the root element is not 'configuration'. Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="ShowError_ConfigUnauthorizedAccess" xml:space="preserve">
+    <value>Can not access NuGet.Config, Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="Unknown_Config_Exception" xml:space="preserve">
+    <value>Got exception From NuGet.Config, Please check NuGet.Config at '{0}'.</value>
+  </data>
+  <data name="UnsupportedDecryptPassword" xml:space="preserve">
+    <value>Decrypt Password For source : '{0}' is unsupported on coreclr, Please use clear text password.</value>
+  </data>
+  <data name="UnsupportedEncryptPassword" xml:space="preserve">
+    <value>Encrypt Password For source : '{0}' is unsupported on coreclr, Please use clear text password.</value>
```

Comment:
Password encryption failed for source : '{0}' is unsupported on this platform. A clear text password may be used instead.